src/pybullet_helpers/link.py

```diff
@@ -0,0 +1,43 @@
+"""Pybullet helper class for link utilities."""
+from typing import NamedTuple
+
+import pybullet as p
+
+from predicators.src.pybullet_helpers.geometry import Pose, Pose3D, \
+    Quaternion, get_pose
+
+BASE_LINK: int = -1
+
+
+class LinkState(NamedTuple):
+    """Link state from Pybullet."""
+    linkWorldPosition: Pose3D
+    linkWorldOrientation: Quaternion
+    localInertialFramePosition: Pose3D
+    localInertialFrameOrientation: Quaternion
+    worldLinkFramePosition: Pose3D
+    worldLinkFrameOrientation: Quaternion
+
+    def com_pose(self) -> Pose:
```

Comment:
property? maybe a functools.cached_property?

Reply:
Same comment as above for `is_circular` in JointInfo - I didn't want to confuse between pybullet data properties and functionality on top. But either way works